.github/workflows/validate-institute-list.yaml

```diff
@@ -0,0 +1,25 @@
+name: Unit tests
+on:
+  push:
+  pull_request:
+
+jobs:
+  run-unit-tests:
+    name: Run unit tests
+    runs-on: ubuntu-latest
+    steps:
+      - name: Check out code
+        uses: actions/checkout@v3
+
+      - name: Set up python
+        uses: actions/setup-python@v4
+        with:
+          python-version: '3.10'
+
+      - name: Install dependencies
+        run: |
+          pip install -r requirements.txt
+
+      - name: Validate institute list
+        run: |
+          python -m process_report.instiute_list_validate --github
```

Comment:
This looks like typo (`instiute_list_validate` should be `institute_list_validate`).

Reply:
Indeed it is :P